### ISSUE TITLE
Project Relicense

### ISSUE THREAD

**Issue by WaveHack** [WaveHack/OpenDominion#366]
@stromblom @Kirbstomper @barmmie @Hodgy @Kender2 @guilhermeportela @straube @MJBrennan @Braunson @gionoa @tommasolevato @ayushpratap @zaiura @srdg @ronanversendaal @mmartinsky @rmikalkenas @mrpudn 

Hi everyone,

I tagged you people because each of you contributed code to the OpenDominion project in the past (no matter how small), and there's something I want to share which deserves your attention:

**OpenDominion is being relicensed from [MIT](https://opensource.org/licenses/MIT) to [GNU AGPLv3](https://www.gnu.org/licenses/agpl-3.0.en.html)**, effective for all code, commits, tags, releases etc, starting today (November 27th, 2018).

## Why the change?

**tl;dr**: So people aren't allowed to steal the code.

When I started this project many years ago, I wasn't too concerned about restrictive open source software licensing, seeing the relatively small scope of the project back then. But now, the project is steadily gaining traction, interest, player popularity and (potential) contributors, and this point has been on my mind for a while.

I care about open source software and sharing improvements, and I believe that using a strong copyleft license is in the best interest of the project, its identity and integrity in the long run, as opposed to the current license.

A slightly longer tl;dr is that people aren't allowed anymore to run away with the code, host their own instance of the the game (potentially monetizing it), without further redistribution of the source code.

The relicensing change will (in theory) ensure that OpenDominion will always be open source with the focus on the community, both players and developers/contributors alike.

## What does this mean for me?

**tl;dr**: Nothing.

In a perfect world I would need explicit written permission from each and every one of you for the relicense, but in practice I really cannot be bothered.

This is why I'm informing you all via this issue and giving you the chance to ask questions, comment, decline etc it via here.

Should anyone of you decline or disagree with this change and/or the new license, please let me know via this issue or in private (email at wavehack dot net, preferably with the reason why), and I shall remove your past contributions and reimplement the removed code myself.

Thus, otherwise you implicitly consent with this change, and no further action is required, other than your brief attention regarding this change.

*Note: I'm not a lawyer or (copyright) law expert. Despite that, I'm trying to play nice with the established rulesets surrounding this. In this particular case, relicensing of open source software. I used this post as a guide for this issue: <https://opensource.stackexchange.com/questions/33/how-can-a-project-be-relicensed/2032#2032>*

Thank you for your time and for your (past) contributions!
